### ISSUE TITLE
Can't download to Chrome browser on Chromebook

### ISSUE THREAD

**Issue by sw-a** [penge/my-notes#145]
I have installed and tried my-notes in the Chrome browser on my PC. It's great -- very handy and well thought out. I would really like to install it on my Acer Spin 11 Chromebook. When I go to the Google Web store for Chromebooks, my-notes shows as installed (it is but only on my Windows PC) and my only option is to uninstall it. No sign of it on my Chromebook. Can't get anywhere with this at the Play store, either, despite the fact some Android apps install and run well on my Chromebook.

I hope my-notes will become available to install in Chromebook Chrome browsers soon.

Thank you and stay safe.

**Comment by penge** [comment]
Hi and thank you. It might be possible Chrome already installed My Notes on your Chromebook as a part of synchronizing the installed extensions across devices if under the same account.

Please try to open the following in a new tab:
```
chrome://extensions
```

It might be possible My Notes is installed and in the list, only asking to be enabled first. Let me know if this worked.

PS: I have Chromebook myself and it was actually my inspiration to create My Notes.

**Comment by sw-a** [comment]
Hi Pavel,

I don't know what happened in the last 24 hours, but you're right! I did look in the extensions list, and my notes wasn't there on my Chromebook. Again later in the evening, I tried again. But just now, after reading your email in the wee hours, my notes was in the extensions list, just as it should be. Maybe a glitch or slowdown in Google's syncing? I don't know, but I am happy to see I've got it now.

Thanks so much for replying.

S,W. Anderson

________________________________

**Comment by sw-a** [comment]
https://www.forbes.com/sites/danalexander/2020/10/16/donald-trump-has-at-least-1-billion-in-debt-more-than-twice-the-amount-he-suggested/

Get Outlook for Android<https://aka.ms/ghei36>

**Comment by penge** [comment]
Interesting, gonna read it.

**Comment by sw-a** [comment]
Uh, red face here. I regularly "share" news items proffered by Google Assistant on my smartphone to myself via e-mail, or to Google Keep, depending on the subject. Yesterday, I saw My Notes was listed with other apps I could use for sharing, so I clicked on that. I expected the link and title would show up in My Notes. When I checked later, it hadn't shown up. Wasn't in the My Notes folder on Google Drive, either. Oh well, I thought.  Now, I see it was shared to here. I had no idea!

I hope you do find the article interesting, but I want to be clear that I wouldn't send a political item to GitHub or any other such site where it well could be deemed completely inappropriate for what the site is about. Hence the red face. :(
